src/cosma/cinterface.cpp

```diff
@@ -0,0 +1,160 @@
+#include <grid2grid/grid2D.hpp>
+#include <grid2grid/ranks_reordering.hpp>
```

Comment:
I guess `grid2grid/ranks_reordering`, `mapper.hpp`, `matrix.hpp`, `context.hpp` and probably `grid2grid/transformer.hpp` are not necessary to include.

Reply:
Correct :)